.github/workflows/someip2val_build.yml

```diff
@@ -56,7 +52,7 @@ jobs:
               cmake g++ build-essential \
               g++-aarch64-linux-gnu binutils-aarch64-linux-gnu \
               jq python3 python3-pip python3-setuptools &&
-          pip3 install -U conan
+          pip3 install conan==1.55.0
```

Comment:
Would it be an idea to add a comment on why we want to pin conan.
I see a risk that sooner or later we will get a vulnerability warning and then it is good to know why we pinned it.

Is it only because newer conan does not support "info"?

Reply:
2.0 broke arm64 cross compiling and also introduce a lot of new recipes, not sure if the state is good enough, but I also fixed the revisions of some important dependencies that only work with specific conan version. It's better to get a security warning than to find some day we are not building at all